File: C_Programming/Day_5/program33.c
/*  
    Description : Write a program that accepts frequency & prints
                  the below pattern : 
    Input : 4 #
    Output : #  # # #
*/

#include<stdio.h>

void Display(int iNo , char ch)
{
    int iCnt = 0;

    for(iCnt = 1 ; iCnt <= iNo ; iCnt++)
    {
        printf("%c\t",ch);
    }

}


int main()
{
    int iValue = 0;
    char ch = '\0';

    printf("Enter the frequency : ");
    scanf("%d",&iValue);

    printf("Enter the character that you want to print : ");
    scanf(" %c",&ch);

    Display(iValue , ch);

    return 0;
}

File: Python_Programming/Day_5/program33.py
'''
    Description : Write a program that accprts frequency & prints
                  the below pattern : 
    Input : 4 #
    Output : #  # # #
'''
def Display(No , ch):

    for i in range(1,No+1,1):
        print(ch , "\t" , end = "")


def main():
    Value = 0
    ch = None

    print("Enter the frequency : ",end = "")
    Value = int(input())

    print("Enter the character that you want to print : ",end = "")
    ch = input()

    Display(Value , ch)

if __name__ == "__main__":
    main()